### ISSUE TITLE
Fuzzy Hashes

### ISSUE THREAD

**Issue by Amerlander** [calliope-edu/CalliopEO_AstroPi#55]
In most test cases we do not need a perfect md5 hash. It mostly does not matter if there is a line more or less.
Changing the hashing from md5 to some fuzzy hash like [`SSDEEP`](https://ssdeep-project.github.io/ssdeep/) or [`sdhash`](http://roussev.net/sdhash/sdhash.html) might help getting less false negative tests.

**Comment by Amerlander** [comment]
Until now I had no issues with the md5 hashes. All errors came from errors in the code or .data files.
At the moment all tests are passing:
```
          Test ended: Wed 24 Feb 23:11:09 UTC 2021

          14 FINISHED
          14 PASSED
          0 NOT PASSED
```

**Comment by rzbrk** [comment]
Or we count the lines in both `.data` files (the `.data` file created during the test and the `.data` file from `./testcases/testfiles/`), take the lowest number (lets name this variable `lines_min`), cut the first `lines_min`lines from both files using the command `head`and compare only this part with `md5sum` or `cmp`. All tools are available on every Linux system.

**Comment by rzbrk** [comment]
I established a bash function `comp()` as described above, see [4fb76aa](https://github.com/calliope-edu/CalliopEO_AstroPi/pull/60/commits/4fb76aa2520637f474fe258fcab0e400af86b18e). The first and second argument of this function are the `.data` files to compare against each other. The third argument is an allowed percentage of lines to differ. If this percentage threshold is not met, the function returns 1 (files differ). If I would not have implemented this feature, a comparison of any `.data` file with an empty file would return 0 (equal).

This issue can be closed if #60 is merged.
